build/application/src/main/resources/dirigible.properties

```diff
@@ -19,4 +19,6 @@ DIRIGIBLE_DATABASE_PROVIDER=local
 DIRIGIBLE_JAVASCRIPT_HANDLER_CLASS_NAME=org.eclipse.dirigible.graalium.handler.GraaliumJavascriptHandler
 DIRIGIBLE_GRAALIUM_ENABLE_DEBUG=true
 DIRIGIBLE_HOME_URL=services/web/ide/
-DIRIGIBLE_DATABASE_NAMES_CASE_SENSITIVE=true
\ No newline at end of file
+DIRIGIBLE_DATABASE_NAMES_CASE_SENSITIVE=true
+DIRIGIBLE_SPRING_ADMIN_USERNAME=admin
```

Comment:
how about security module used is not the basic one (used for demo only)

Reply:
We can test it if you have configured such a system.
In production cases, you are going to use (Spring Cloud Discovery)[https://docs.spring-boot-admin.com/current/getting-started.html#discover-clients-via-spring-cloud-discovery].